cloudformation/alarms.yaml

```diff
@@ -491,3 +571,27 @@ Resources:
       Period: 60
       Statistic: Maximum
       Threshold: 0
+      OKActions:
+        - !Ref CloudWatchAlarmSNSTopic
+      AlarmActions:
+        - !Ref CloudWatchAlarmSNSTopic
+  CloudWatchAlarmSNSTopic:
+    Type: AWS::SNS::Topic
+    Properties: 
+      TopicName: !Join ['-', [FhirSolution, !Ref Stage, cloudwatch, alarm, topic]]
```

Comment:
We have been trying to avoid using direct names to enable better upgrade paths.

> Important: If you specify a name, you can't perform updates that require replacement of this resource. You can perform updates that require no or some interruption. If you must replace the resource, specify a new name. https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-sns-topic.html

I believe if you remove this `name` it will generate an SNS topic name that makes sense (using stack name); could you test that out